libraries/AP_Landing/AP_Landing.h

```diff
@@ -134,6 +134,10 @@ class AP_Landing {
     adjusted_relative_altitude_cm_fn_t adjusted_relative_altitude_cm_fn;
     disarm_if_autoland_complete_fn_t disarm_if_autoland_complete_fn;
     update_flight_stage_fn_t update_flight_stage_fn;
+    
+    // saved bearing for yaw correction after touchdown
+    float runway_bearing;
+    bool touched_down;
```

Comment:
can you move touched_down up into the flags struct?

Reply:
Done